packages/nextjs/next.config.js

```diff
@@ -1,4 +1,6 @@
 // @ts-check
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const { withPlausibleProxy } = require("next-plausible");
```

Comment:
:(

![image](https://github.com/BuidlGuidl/abi.ninja/assets/2486142/f6c11b3d-6022-482b-8eb1-828820ca0897)

Reply:
Ohh yes encountered this earlier, to handle it we can convert `next.config.js` => [`next.config.mjs`](https://nextjs.org/docs/pages/api-reference/next-config-js) and then we can use `import` and export default : 

```js
// @ts-check
import { withPlausibleProxy } from "next-plausible";

/** @type {import('next').NextConfig} */
const nextConfig = {
  ...
};

export default withPlausibleProxy()(nextConfig);
```

But yeah even ignoring is completely fine I feel

Reply:
Changed to `.mjs` to use ESM :)

Thanks!

https://github.com/BuidlGuidl/abi.ninja/pull/47/commits/ba264a6f9b2830e2b565cea8dd9de2366ba5b9dc